scripts/deploy-bot-layer.sh

```diff
@@ -41,7 +47,7 @@ zip -r -q medplum-bot-layer.zip .
 
 # Publish the bot layer
 aws lambda publish-layer-version \
-  --region us-east-1 \
+  --region $AWS_REGION \
```

Comment:
We should just remove this line entirely.  The AWS CLI tools will pull the default value the exact same way `aws configure get region` does.  And that would also allow users to specify on the command line directly.

```bash
AWS_REGION=us-east-2 ./deploy-bot-layer.sh
```